### PR TITLE
dev/core1329 Reduce number of deceased contacts in the demo data from…

### PR DESCRIPTION
**Pull request by seamuslee001**
… 31 to 12

Overview
----------------------------------------
This reduces the number of deceased persons in the sample data from currently on dmaster 31 to 12

ping @jusfreeman @eileenmcnaughton

### CODE REVIEW

File: sql/GenerateData.php [hunk 1]
```diff
@@ -649,7 +649,7 @@ private function addIndividual() {
       }
 
       // Deceased probability based on age
-      if ($age > 40) {
+      if ($age > 60) {
```
Comment (jusfreeman) [inline sql/GenerateData.php]: Thanks @seamuslee001 

Can we try this, if male then 64 years and if female then 68 years.

Refs:
https://www.hcf.com.au/health-agenda/health-care/research-and-insights/life-expectancy-in-australia
https://www.qgso.qld.gov.au/statistics/theme/population/vital-statistics/life-expectancy#current-release-life-expectancy-birth-years-sex-qld-australia

**Comment by jusfreeman** [comment]
"Two (deceased contacts) is more than enough to illustrate the concept."
From https://lab.civicrm.org/dev/core/issues/1329#note_25149

Related Gitlab, https://lab.civicrm.org/dev/core/issues/1329

**Comment by seamuslee001** [comment]
ok @jusfreeman have implemented those changes now

**Review by jusfreeman — APPROVED** [review]
@seamuslee001 thanks, looks good to me. With this change we will have at most 4 deceased contacts in the demo data, that's a definite improvement!

**Comment by seamuslee001** [comment]
Merging as per Justin's review

**Pull request merged**